tests/ci/build_report_check.py

```diff
@@ -228,6 +231,7 @@ def main():
     total_groups = len(build_results)
     logging.info("Totally got %s artifact groups", total_groups)
     if total_groups == 0:
+        fail_simple_check(gh, pr_info, f"{build_check_name} failed")
```

Comment:
nnnnnaaaaaaah... I don't think so =)

![image](https://user-images.githubusercontent.com/3025537/180264280-40d5e6d4-5bea-4498-b1d7-69ec5899f267.png)

`BuildReport*` run always. Even on the `do not test` label. So if there aren't builds, we won't merge it ever.